edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Program.cs

```diff
@@ -66,6 +66,7 @@ static async Task<int> MainAsync(IConfigurationRoot configuration)
             LogLogo(logger);
             LogVersionInfo(logger);
             logger.LogInformation($"OptimizeForPerformance={configuration.GetValue("OptimizeForPerformance", true)}");
+            logger.LogInformation($"MessageAckTimeoutSecs={configuration.GetValue("MessageAckTimeoutSecs", 30)}");
```

Comment:
I followed the same pattern for OptimizeForPerformance, but this seems error prone as we specify the defaults here as well as DependencyManager.cs. We should explore cleaning this up somehow in the future.